### PR TITLE
Makefile workflow

### PR DESCRIPTION
**Pull request by SharafatKarim**
Automatically testing solution whenever anyone does some kind of push or pull request.

### CODE REVIEW

**Comment by SharafatKarim** [comment]
@sr-tamim @ahmubashshir bro review please!

**Review by SharafatKarim — COMMENTED** [review]
Nothing to say

File: .github/workflows/cpp-makefile.yml
Comment (ahmubashshir) [inline .github/workflows/cpp-makefile.yml:1]: LGTM

**Review by ahmubashshir — COMMENTED** [review]
LGTM

**Review by sr-tamim — APPROVED** [review]

**Pull request merged**